astroid/helpers.py

```diff
@@ -20,6 +20,10 @@
 from astroid.nodes import scoped_nodes
 from astroid.typing import InferenceResult
 
+# Don't remove the following safe_infer import without a deprecation notice.
```

Comment:
Do we want to do this now? What's the rationale for not doing it immediately?

Reply:
f598696